File: app/src/main/java/me/tysheng/xishi/adapter/TyBindingViewHolder.java
package me.tysheng.xishi.adapter;

import android.databinding.ViewDataBinding;
import android.view.View;

import com.chad.library.adapter.base.BaseViewHolder;

/**
 * Created by tysheng
 * Date: 2017/5/11 14:42.
 * Email: tyshengsx@gmail.com
 */

public class TyBindingViewHolder<Binding extends ViewDataBinding> extends BaseViewHolder {
    private Binding mBinding;

    public TyBindingViewHolder(View view) {
        super(view);
    }

    public Binding getBinding() {
        return mBinding;
    }

    public void setBinding(Binding binding) {
        mBinding = binding;
    }
}

File: app/src/main/java/me/tysheng/xishi/adapter/TyBindingViewHolder.kt
package me.tysheng.xishi.adapter

import android.databinding.ViewDataBinding
import android.view.View

import com.chad.library.adapter.base.BaseViewHolder

/**
 * Created by tysheng
 * Date: 2017/5/11 14:42.
 * Email: tyshengsx@gmail.com
 */

class TyBindingViewHolder<Binding : ViewDataBinding>(view: View) : BaseViewHolder(view) {
    var binding: Binding? = null
}
